WeatherService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Weather_Monitoring_System.Bots;
+using Weather_Monitoring_System.DataParsers;
+using Weather_Monitoring_System.Models;
+
+namespace Weather_Monitoring_System
+{
+    public class WeatherService
```

Comment:
Why you don't have an interface for this class?

Reply:
Why I must add an interface? I added this class to make an object from all the interfaces I have and I didn't think in the future I would need to add another class like this

Reply:
The thing you should come up with after finishing the SOLId topic is to represent each and every thing in terms of interfaces. 
You might do something in the future to have another implementation. 
Furthermore, utilizing interfaces is a crucial aspect when implementing other SOLID principles.